### PR TITLE
Renamed warp behaviors and spawn types

### PR DESCRIPTION
**Pull request by coleferg**
The two tables in area.c are used to match the behavior with a spawn type. The index of the behavior in the first table is used to return a spawn type from the second table. A lot more renaming could probably be done by looking in `level_update.c` `set_mario_initial_action` and matching the spawn type from mario's act. 

I changed the formatting of the behavior table (now 4 columns) so that it matches the spawn type table, this way its easier to see the connection between these two tables.

### CODE REVIEW

File: src/game/area.c [hunk 1]
```diff
@@ -120,8 +131,8 @@ u32 get_mario_spawn_type(struct Object *o) {
     const BehaviorScript *behavior = virtual_to_segmented(0x13, o->behavior);
 
     for (i = 0; i < 20; i++) {
-        if (D_8032CE9C[i] == behavior) {
-            return D_8032CEEC[i];
+        if (WARP_BEHAVIOR_SPAWN_TABLE[i] == behavior) {
+            return SPAWN_TYPE_FROM_WARP_BHV[i];
```
Comment (nim-ka) [inline src/game/area.c]: These variable names should be camelCase; UPPER_SNAKE_CASE denotes a macro

**Comment by coleferg** [comment]
sounds good, I'll change it

File: src/game/area.c (hunk 1 above)
Comment (coleferg) [reply src/game/area.c]: sounds good, I'll change it

Comment (coleferg) [reply src/game/area.c]: These vars have been camel cased!

Comment (coleferg) [reply src/game/area.c]: and now prefixed with "s"

**Pull request closed without merging**